modules/api/src/main/java/org/apache/ignite/configuration/schemas/metastorage/MetastoreManagerConfigurationSchema.java

```diff
@@ -1,7 +1,8 @@
-package org.apache.ignite.metastorage.configuration;
+package org.apache.ignite.configuration.schemas.metastorage;
 
 import org.apache.ignite.configuration.annotation.ConfigurationRoot;
 import org.apache.ignite.configuration.annotation.Value;
+import org.apache.ignite.configuration.schemas.TempConfigurationStorage;
 
 @ConfigurationRoot(rootName = "metastore", storage = TempConfigurationStorage.class)
 public class MetastoreManagerConfigurationSchema {
```

Comment:
I'd rather say that it's ClusterConfigurationSchema with DISTRIBUTED configurationType. What do you think?

Reply:
Renamed.